invenio_app_rdm/theme/views.py

```diff
@@ -67,3 +67,9 @@ def can_list_files(record):
     """
     PermissionPolicy = get_record_permission_policy()
     return PermissionPolicy(action='read_files', record=record).can()
+
+
+@blueprint.route('/search/')
```

Comment:
doubt: the previous does not have the final `/` and sometimes flask goes bananas with added/missing slashes.... tests this and make it consistent (ignore the comment if it is)